sources/packages/backend/apps/db-migrations/src/sql/StudentAssessments/Update-student-assessment-status.sql

```diff
@@ -0,0 +1,40 @@
+-- Updates all assessments associated with active applications.
+UPDATE
+  sims.student_assessments sa
+SET
+  student_assessment_status = (
+    CASE
+      WHEN sa.assessment_workflow_id IS NULL THEN 'Submitted'
```

Comment:
Submitted is default status of the column, should we need a condition for that?

Reply:
I remove it. There is just a small drawback related to garbage data that can exit on local DBs but I did not consider that a good reason to keep it.